### PR TITLE
American medical association changes

### PR DESCRIPTION
**Pull request by serpodrick**
Changes for american-medical-association and friends, as indicated in the commit messages.

http://validator.citationstyles.org/ hasn't seemed to be working for at least the last few days: "Validation is taking longer than expected! (more than 6 seconds)". I'm hoping Travis has more patience.

### CODE REVIEW

**Comment by rmzelle** [comment]
The validator works for me (test e.g. http://validator.citationstyles.org/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fcitation-style-language%2Fstyles%2Fmaster%2Fapa.csl&version=1.0 ).

**Comment by rmzelle** [comment]
For easier localization, it would also be better to use 

```
<text term="volume" form="short" strip-periods="true"/>
<text variable="volume" prefix=" " suffix="."/>
```

instead of `<text variable="volume" prefix="Vol " suffix="."/>`

https://github.com/citation-style-language/locales/blob/master/locales-en-US.xml#L191

**Comment by rmzelle** [comment]
And instead of `prefix=" Vol "`, it would be better to place the delimiter on a `<group/>` element. E.g. you can rewrite

```
        <else-if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
          <group prefix=" ">
            <text term="in" text-case="capitalize-first" suffix=": "/>
            <text macro="editor"/>
            <text variable="container-title" font-style="italic" prefix=" " suffix="." text-case="title"/>
            <text variable="volume" prefix=" Vol " suffix="."/>
            <text macro="edition" prefix=" "/>
            <text variable="collection-title" prefix=" " suffix="."/>
            <group suffix=".">
              <text macro="publisher" prefix=" "/>
              <group suffix="." prefix="; ">
                <date variable="issued">
                  <date-part name="year"/>
                </date>
                <text variable="page" prefix=":"/>
              </group>
            </group>
          </group>
        </else-if>
```

to

```
        <else-if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
          <group prefix=" " delimiter=" ">
            <text term="in" text-case="capitalize-first" suffix=":"/>
            <text macro="editor"/>
            <text variable="container-title" font-style="italic" suffix="." text-case="title"/>
            <text variable="volume" prefix="Vol " suffix="."/>
            <text macro="edition"/>
            <text variable="collection-title" suffix="."/>
            <group suffix=".">
              <text macro="publisher"/>
              <group suffix="." prefix="; ">
                <date variable="issued">
                  <date-part name="year"/>
                </date>
                <text variable="page" prefix=":"/>
              </group>
            </group>
          </group>
        </else-if>
```

**Comment by serpodrick** [comment]
There you go.

Your link to the validator worked the first time for me (though it found 9 errors not of my making!), but after that it stopped working. I'm not sure what the problem is, so maybe it's on my end.

**Comment by rmzelle** [comment]
@serpodrick, sorry, can't reproduce any issues with the validator. The link I sent intentionally produced some errors (it's validating the current version of apa.csl against the old 1.0 schema). Producing validation errors is a good way to make sure validation works correctly. :)

@adam3smith, happy with this?

**Comment by adam3smith** [comment]
I'll check tonight & will merge if there are no concerns from me.

File: american-medical-association-alphabetical.csl [hunk 1]
```diff
@@ -211,9 +213,19 @@
           <group prefix=" " suffix=".">
             <text variable="container-title" font-style="italic" form="short"/>
             <group delimiter=";" prefix=" ">
-              <date variable="issued">
-                <date-part name="year"/>
-              </date>
+             <choose>
```
Comment (adam3smith) [inline american-medical-association-alphabetical.csl]: I don't agree with your reading of the manual here. I think, if you look at the example, they're referring to journal articles with neither volume nor issue. So in line 217 you should have ```variable="issue volume"``` instead.

**Comment by adam3smith** [comment]
Apart from that one comment, everything looks great, thanks!

**Comment by serpodrick** [comment]
Committed.

**Comment by rmzelle** [comment]
Great, thanks Pod!

**Pull request merged**